test/dataframe.jl

```diff
@@ -51,17 +51,7 @@ module TestDataFrame
     @test names(dfc[1, :b]) == [:c, :e]
     @test names(dfdc[1, :b]) == [:c]
 
-    #
-
     x = DataFrame(a = [1, 2, 3], b = [4, 5, 6])
-    v = DataFrame(a = [5, 6, 7], b = [8, 9, 10])
-
-    z = vcat(v, x)
-
-    # TODO: uncomment the line below after deprecation
-    # @test_throws ArgumentError z[:, [1, 1, 2]]
```

Comment:
Why this line (61 and 62) is removed? I think we should test that it throws an error after deprecation.

Reply:
Yes, but currently this is not fully deprecated behavior so it does not yet throw an error.  When automatic generation of column names from indexing is fully deprecated, this line should be un-commented.  Note that we test the current behavior in `deprecation.jl`.